website/addons/base/views.py

```diff
@@ -202,20 +192,21 @@ def get_auth(auth, **kwargs):
 
     try:
         credentials = provider_settings.serialize_waterbutler_credentials()
-        settings = provider_settings.serialize_waterbutler_settings()
+        waterbutler_settings = provider_settings.serialize_waterbutler_settings()
     except exceptions.AddonError:
         log_exception()
         raise HTTPError(httplib.BAD_REQUEST)
 
-    return {
+    return jwt.encode({
+        'exp': datetime.datetime.utcnow() + datetime.timedelta(seconds=settings.WATERBUTLER_JWT_EXPIRATION),
```

Comment:
This is very unlikely but I'll at the least post the question. Any concern for key collision, should we nest our data under `data`? https://github.com/jpadilla/pyjwt#support-of-registered-claim-names